destination/destination.go

```diff
@@ -0,0 +1,213 @@
+package destination
```

Comment:
noticed that we don't have our usual files header, so we need to add the header linter and the license file